### ISSUE TITLE
Relationships are not generating

### ISSUE THREAD

**Issue by SuperlativeEntity** [BioDesignRealWorld/LaravelDatabaseDesigner#3]
![relation](https://cloud.githubusercontent.com/assets/3331835/4632635/a4103450-53bf-11e4-88cf-fb0fa01012ac.png)

Relational code not generating

php artisan generate:migration create_clients_table --fields="reference:string:length('30'):unique,client_type_id:integer:unsigned:index,"

php artisan generate:migration create_client_types_table --fields="description:string,"

**Comment by hhsadiq** [comment]
Yes the relationships are not being generated....This tool will be useless without this feature.

**Comment by bilogic** [comment]
where does `generate:migration` comes from?
